sdk/storage/azure-storage-blob/samples/blob_samples_common_async.py

```diff
@@ -10,43 +10,21 @@
 import asyncio
 from azure.core.exceptions import ResourceExistsError
 
-try:
-    import settings_real as settings
-except ImportError:
-    import blob_settings_fake as settings
-
-from testcase import (
-    StorageTestCase,
-    TestMode,
-    record
-)
 
+# set up
 SOURCE_FILE = 'SampleSource.txt'
+data = b"Lorem ipsum dolor sit amet, consectetur adipiscing elit"
```

Comment:
Why don't we have a sample file in the repo rather than writing it on execution?